setup_wifi.py

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env python3
+
+# Script for changing MultiAP WiFi settings in Teltonika RUTX11.
+# Requires RUTX11 device with configuration set by first_setup.sh script - by default done by Husarion during production.  
+
+import click
```

Comment:
```suggestion
from click import secho
```
Lets import only necessary methods